docs/source/getting-started/install.md

```diff
@@ -206,7 +206,7 @@ execute `docker run` will be use to persist the backend server data.
 
 If you are somewhat familiar with Python development, you can also install Plone locally
 without using Docker. Check the [backend configuration](../configuration/backend.md) section.
-It also has more information on plone.volto.
+It also has more information on Plone.Volto.
```

Comment:
Revert. If Vale complains about this term, then add it to `accept.txt`.

Also the original author should have used an inline literal for this, which I think will avoid a Vale warning. Thus:

```suggestion
It also has more information on `plone.volto`.
```